docs/development/releasing.rst

```diff
@@ -26,10 +26,10 @@ packages that use the full bugfix/maintenance branch approach.)
    addresses.  Look for any mis-named entries or duplicates, and add them to the
    ``.mailmap`` file (matched to the appropriate canonical name/email address.)
    Once you have finished this, you can could the number of lines in
-   ``git shortlog -s`` to get the final contributor count.  Also be sure to
-   use the names in that list to update the ``docs/credits.rst`` file. (The
-   ``author_lists.py`` script in the `astropy-tools repository`_ helps with
-   this.)
+   ``git shortlog -s`` to get the final contributor count.
+#. Also be sure to update the ``docs/credits.rst`` file to include any new
```

Comment:
is this step needed for bugfixes? if not, could you add the ``(Only for major versions)`` note here, too?